tests/test_buffer_dynamic.py

```diff
@@ -224,28 +233,34 @@ def test_changeCableLen(self, dvs, testlog):
         self.app_db.wait_for_entry("BUFFER_PROFILE_TABLE", expectedProfile)
         self.app_db.wait_for_field_match("BUFFER_PG_TABLE", "Ethernet0:3-4", {"profile": "[BUFFER_PROFILE_TABLE:" + expectedProfile + "]"})
 
-        # remove lossless PG 3-4 on interface
+        # Remove lossless PG 3-4 on interface
         self.config_db.delete_entry('BUFFER_PG', 'Ethernet0|3-4')
 
+        # Shutdown interface
+        dvs.runcmd('config interface shutdown Ethernet0')
+
```

Comment:
Now that each function start a port and shutdown it on the last, should you add validation if it is running as expected before the test cont?
I see you have a dedicated function for shutdown which is great, the question if if on each test case you would like to add the proper validation as well.

Reply:
I think we don't need that because when the vsimage starts there is no PG configured. So
- at the beginning, we start the port and then configure the PG; 
- at the end, we remove the PG and then shutdown the port. this is to restore the environment.